demo/top_down_full_img_video_demo.py

```diff
@@ -0,0 +1,111 @@
+import os
```

Comment:
What does the "full" in filename mean?

Reply:
No object detection model used. 
Directly input the full size image (without bbox cropping) into the model.

Reply:
"img_video" means img or video, or means "full_img" version of "video"

Reply:
Aha yes, "full_img" version of "video".

Reply:
How about "top_down_video_demo_full_frame_without_det"